src/java/htsjdk/tribble/dbsnp/OldDbSNPCodec.java

```diff
@@ -89,6 +89,11 @@ public OldDbSNPFeature decode(String line) {
     }
 
     @Override
+    public boolean canDecode(String path){
+	return path.toLowerCase().endsWith(".dbsnp") || path.toLowerCase().endsWith(".rod") || path.toLowerCase().endsWith(".snp");
```

Comment:
Only if they are using a toolkit that relies on `canDecode()` for the selection of codecs for Tribble-supported file formats. Hellbender is the only such toolkit, as far as I know, so they will only be required to use the ".dbsnp" extension if they want to read a dbsnp file in hellbender. Tools that don't try to do fancy auto-selection of codecs will be unaffected.